### PR TITLE
[barefoot][platform] Update BFN platforms

### PR DESCRIPTION
**Pull request by vsenchyshyn**
Signed-off-by: Vitaliy Senchyshyn <vsenchyshyn@barefootnetworks.com>

**- Why I did it**
Added support of BFN newport new platform name.
Updated debian version for montara platform.

**- How I did it**

**- How to verify it**

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/barefoot/platform-modules-bfn-newport.mk [hunk 1]
```diff
@@ -4,8 +4,17 @@ BFN_NEWPORT_PLATFORM_MODULE_VERSION = 1.0
...
 $(BFN_NEWPORT_PLATFORM_MODULE)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-modules-bfn-newport
 $(BFN_NEWPORT_PLATFORM_MODULE)_DEPENDS += $(LINUX_HEADERS) $(LINUX_HEADERS_COMMON)
-$(BFN_NEWPORT_PLATFORM_MODULE)_PLATFORM = x86_64-accton_as9516bf_32d-r0
+$(BFN_NEWPORT_PLATFORM_MODULE)_PLATFORM = x86_64-accton_as9516_32d-r0
 SONIC_DPKG_DEBS += $(BFN_NEWPORT_PLATFORM_MODULE)
+
+BFN_NEWPORT_BF_PLATFORM_MODULE = sonic-platform-modules-bfn-newport-as9516bf_$(BFN_NEWPORT_PLATFORM_MODULE_VERSION)_amd64.deb
+$(BFN_NEWPORT_BF_PLATFORM_MODULE)_PLATFORM = x86_64-accton_as9516bf_32d-r0
+$(eval $(call add_extra_package,$(BFN_NEWPORT_PLATFORM_MODULE),$(BFN_NEWPORT_BF_PLATFORM_MODULE)))
+#$(BFN_NEWPORT_BF_PLATFORM_MODULE)_SRC_PATH = $(PLATFORM_PATH)/sonic-platform-modules-bfn-newport
```
Comment (akokhan) [inline platform/barefoot/platform-modules-bfn-newport.mk]: please remove if not needed

File: platform/barefoot/sonic-platform-modules-bfn-newport/debian/rules [hunk 2]
```diff
@@ -17,14 +18,16 @@ override_dh_auto_build:
...
 override_dh_auto_install:
-	dh_installdirs -p$(PACKAGE_NAME) $(KERNEL_SRC)/$(INSTALL_MOD_DIR)
-	cp $(MODULE_SRC)/*.ko debian/$(PACKAGE_NAME)/$(KERNEL_SRC)/$(INSTALL_MOD_DIR)
-	dh_installdirs -p$(PACKAGE_NAME) usr/local/bin
-	cp -r $(SCRIPT_SRC)/* debian/$(PACKAGE_NAME)/usr/local/bin
-	dh_installdirs -p$(PACKAGE_NAME) etc/network/interfaces.d/
-	cp -r $(CONFIGS_SRC)/network/interfaces.d/* debian/$(PACKAGE_NAME)/etc/network/interfaces.d/
-	dh_installdirs -p$(PACKAGE_NAME) usr/share/sonic/device/x86_64-accton_as9516bf_32d-r0/plugins
-	cp -r $(PLUGINS_DIR)/*  debian/$(PACKAGE_NAME)/usr/share/sonic/device/x86_64-accton_as9516bf_32d-r0/plugins/
+	(for mod in $(MODULE_NAMES); do \
```
Comment (akokhan) [inline platform/barefoot/sonic-platform-modules-bfn-newport/debian/rules:21]: Can't we install one package and just use post install step to create sym link?

Comment (vsenchyshyn) [reply platform/barefoot/sonic-platform-modules-bfn-newport/debian/rules:21]: The problem is that the deb package is strictly associated with the specific platform and won't be even installed for another one, so there is no chance for such link creation without making some kind of hack in SONiC build system.

**Comment by akokhan** [comment]
LGTM

**Pull request closed without merging**